rollup.config.js

```diff
@@ -1,10 +1,24 @@
 import buble from 'rollup-plugin-buble'
+import nodeResolve from 'rollup-plugin-node-resolve'
+import pkg from './package.json'
 
 export default {
-  entry: 'src/index.js',
-  dest: 'dist/hold.js',
-  moduleName: 'mostHold',
-  format: 'umd',
-  sourceMap: true,
-  plugins: [buble()]
+  input: 'src/index.js',
+  plugins: [
+    buble(),
+    nodeResolve()
```

Comment:
Hm.. why do you want to bundle dependency?

Reply:
What are the pros and cons of bundling them vs not?

Reply:
You will have possibly not tree shakable duplication in your app. `instanceof` will be broken because it refers to different classes. You should distribute plugin itself, not whole library which will be used separatly by user.

Reply:
Right.  Are there any downsides, for example, for non-ES module users?  How would you configure the UMD rollup build for this case?  Could you provide an example?

Thanks!

Reply:
Globals option let's to specify aliases for `window` formats.
```js
export default {
  input: 'index.js',
  output: [
    {
      format: 'umd',
      name: 'mostHold',
      globals: {
        '@most/core': 'mostCore'
      },
      file: 'dist/index.js'
    },
    {
      format: 'es',
      file: 'dist/index.es.js'
    }
  ]
};
```

Reply:
:+1: That looks good to me.  Anyone have any concerns about going this route?

Reply:
I just pushed d3afcd5, which omits deps from the build.  Take a look and see what you think.